mailu/values.yaml

```diff
@@ -1240,6 +1240,11 @@ postfix:
   ##     my_variable = my_value
   overrides: {}
 
+  ## @param postfix.securityContext `securityContext` for postfix container
+  securityContext: {}
+  ## @param postfix.securityContext `securityContext` for postfix pod
```

Comment:
replace `postfix.securityContext` with `postfix.podSecurityContext`

Reply:
Good catch, should be fixed now!
I've also rebased to latest `master`.